src/nemos/basis/_basis.py

```diff
@@ -981,8 +986,8 @@ def split_by_feature(
         >>> split_features = basis.split_by_feature(X, axis=1)
         >>> for feature, arr in split_features.items():
         ...     print(f"{feature}: shape {arr.shape}")
-        feature_1: shape (20, 1, 5)
-        feature_2: shape (20, 1, 6)
+        feature_1: shape (20, 5)
+        feature_2: shape (20, 6)
         >>> # If one of the basis components accepts multiple inputs, the resulting dictionary will be nested:
```

Comment:
```suggestion
        >>> # split_by_features can handle multiple inputs as well
```
I think this line is out of date, right?